iidm/iidm-criteria/src/main/java/com/powsybl/iidm/criteria/SingleCountryCriterion.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.criteria;
+
+import com.google.common.collect.ImmutableList;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.criteria.translation.NetworkElement;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Etienne Lesot {@literal <etienne.lesot@rte-france.com>}
+ */
+public class SingleCountryCriterion implements Criterion {
+
+    private final List<Country> countries;
+
+    public SingleCountryCriterion(List<Country> countries) {
+        Objects.requireNonNull(countries);
+        this.countries = ImmutableList.copyOf(countries);
+    }
+
+    @Override
+    public CriterionType getType() {
+        return CriterionType.SINGLE_COUNTRY;
+    }
+
+    @Override
+    public boolean filter(Identifiable<?> identifiable, IdentifiableType type) {
```

Comment:
Why lines, tie lines, VSC, LCC and HVDC lines are false?

Reply:
Lines, tie lines and HVDC lines are managed by `TwoCountryCriterion`.

I added the VSC and LCC.